commands/files/serialfile.go

```diff
@@ -84,10 +84,25 @@ func (f *serialFile) NextFile() (File, error) {
 	// open the next file
 	fileName := fp.Join(f.name, stat.Name())
 	filePath := fp.Join(f.path, stat.Name())
+	st, err := os.Lstat(filePath)
```

Comment:
alright, i added support for straight up 'adding' a symlink by itself. Then i went to add a `--dereference` flag, and ran into issues. The deref flag needs to somehow get passed into `commands/cli/parse.go` so we can choose whether or not to follow the symlink.